Sources/SQLKit/Expressions/Queries/SQLCreateTrigger.swift

```diff
@@ -58,133 +124,179 @@ public struct SQLCreateTrigger: SQLExpression {
 
         serializer.statement {
             $0.append("CREATE")
-            if syntax.contains(.supportsConstraints), self.isConstraint { $0.append("CONSTRAINT") }
+            if syntax.contains(.supportsConstraints), self.isConstraint {
+                $0.append("CONSTRAINT")
+            }
+            if let definer = self.definer, syntax.contains(.supportsDefiner) {
+                $0.append("DEFINER =", definer)
+            }
             $0.append("TRIGGER", self.name)
-            $0.append(self.when)
-            $0.append(self.event)
-            if let columns = self.columns, !columns.isEmpty, syntax.contains(.supportsUpdateColumns) { $0.append("OF", SQLList(columns)) }
+            $0.append(self.when, self.event)
+            if let columns = self.columns, !columns.isEmpty, syntax.contains(.supportsUpdateColumns) {
+                $0.append("OF", SQLList(columns))
+            }
+            
             $0.append("ON", self.table)
-            if let referencedTable = self.referencedTable, syntax.contains(.supportsConstraints) { $0.append("FROM", referencedTable) }
-            if let timing = self.timing, syntax.contains(.supportsConstraints) { $0.append(timing) }
+            
+            if let referencedTable = self.referencedTable, syntax.contains(.supportsConstraints) {
+                $0.append("FROM", referencedTable)
+            }
+            
+            if let timing = self.timing, syntax.contains(.supportsConstraints) {
+                $0.append(timing)
+            }
+            
             if syntax.contains(.requiresForEachRow) || (syntax.isSuperset(of: [.supportsForEach, .supportsConstraints]) && self.isConstraint) {
-                $0.append(SQLTriggerEach.row)
+                $0.append(EachSpecifier.row)
             } else if syntax.contains(.supportsForEach), let each = self.each {
                 $0.append(each)
             }
+            
             if let condition = self.condition, syntax.contains(.supportsCondition) {
                 $0.append("WHEN", syntax.contains(.conditionRequiresParentheses) ? SQLGroupExpression(condition) : condition)
             }
+            
             if let order = self.order, let orderTriggerName = self.orderTriggerName, syntax.contains(.supportsOrder) {
-                $0.append(order)
-                $0.append(orderTriggerName)
+                $0.append(order, orderTriggerName)
             }
+
             if syntax.contains(.supportsBody), let body = self.body {
-                $0.append("BEGIN")
-                $0.append(SQLList(body, separator: SQLRaw(" ")))
-                $0.append("END;")
+                $0.append("BEGIN", SQLList(body, separator: SQLRaw(" ")), "END;")
             } else if let procedure = self.procedure {
                 $0.append("EXECUTE PROCEDURE", procedure)
             }
         }
     }
 }
 
-public enum SQLTriggerWhen: SQLExpression {
```

Comment:
Is this breaking?

Reply:
No; the name is typealiased over in `SQLExpressions+Deprecated.swift`